Components/navbar/navbar.css

```diff
@@ -23,17 +24,56 @@
 }
 
 .hamburger {
+  display: block;
   cursor: pointer;
   font-size: 2rem;
 }
 
+.nav-links {
+  position: absolute;
+  left: 0;
+  right: 0;
+  /* top: -180px; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 8rem;
+  background-color: var(--primary);
+  z-index: 1;
+  transition: 0.3s all;
+}
+
+.nav-links a {
+  margin: 0 1rem;
+}
+
+.nav-links-not-visible {
+  top: -180px;
```

Comment:
I think translate could be a better alternative here. 
read more about it - https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

Reply:
Made the required changes.